### PR TITLE
Updating ReadMEs for Jaeger, Zipkin, OTLP exporters

### PR DESCRIPTION
**Pull request by Austin-Tan**
One thought: Does the ReadMe look too cluttered this way? We are listing every option available in configuring the exporter. The descriptions are brief, but it is still a bullet point of 5-ish entries each. @reyang  @cijothomas  @eddynaka please advise.

## Changes

#1584  #1587 #1504  all added the batch exporter and exporter type properties. We include these in the ReadMEs.

#1557 and #1572 removed the ServiceName option from Otlp and Jaeger Exporters. We removed these from ReadMes.

For significant contributions please make sure you have completed the following items:

* [ ] `CHANGELOG.md` updated for non-trivial changes
* [ ] Design discussion issue #
* [ ] Changes in public API reviewed

### CODE REVIEW

File: src/OpenTelemetry.Exporter.OpenTelemetryProtocol/README.md [hunk 1]
```diff
@@ -18,13 +18,17 @@ dotnet add package OpenTelemetry.Exporter.OpenTelemetryProtocol
...
-You can configure the `OtlpExporter` by following the directions below:
+You can configure the `OtlpExporter` through `OtlpExporterOptions` properties:
 
-* `ServiceName`: Name of the service reporting telemetry.
 * `Endpoint`: Target to which the exporter is going to send traces or metrics.
 * `Credentials`: Client-side channel credentials.
 * `Headers`: Optional headers for the connection.
 * `ChannelOptions`: gRPC channel options.
+* `ExportProcessorType`: Whether the exporter should use
+  [Batch or Simple base exporter](https://github.com/open-telemetry/opentelemetry-dotnet/tree/master/src/OpenTelemetry#introduction)
```
Comment (cijothomas) [inline src/OpenTelemetry.Exporter.OpenTelemetryProtocol/README.md:28]: pehaps link to the spec : https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/trace/sdk.md#built-in-span-processors

**Comment by cijothomas** [comment]
> One thought: Does the ReadMe look too cluttered this way? We are listing every option available in configuring the exporter. The descriptions are brief, but it is still a bullet point of 5-ish entries each.

Agree. We can have docs explain the concepts, and also explain options which requires some explanation. And then simply link to the actual class file for the full list of option. (Lets address this separately, as we need to do it for pretty much all instrumentation)

File: src/OpenTelemetry.Exporter.OpenTelemetryProtocol/README.md (hunk 1 above)
Comment (Austin-Tan) [reply src/OpenTelemetry.Exporter.OpenTelemetryProtocol/README.md:28]: Ah I linked to the API intro where it mentioned those but this is a better, more guided and specific reference. Thanks I'll update it.

Comment (cijothomas) [reply src/OpenTelemetry.Exporter.OpenTelemetryProtocol/README.md:28]: Most of the time, linking to spec is right, as its our source of truth! For anything specific to .NET implementation (this repo), we should link to this repo itself.

File: src/OpenTelemetry.Exporter.Zipkin/README.md [hunk 2]
```diff
@@ -26,6 +26,11 @@ in `ZipkinExporterOptions`:
    sending to Zipkin (default false).
 * `MaxPayloadSizeInBytes`: Maximum payload size - for .NET versions
    **other** than 4.5.2 (default 4096).
+* `ExportProcessorType`: Whether the exporter should use
+  [Batch or Simple base exporter](https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/trace/sdk.md#built-in-span-processors)
```
Comment (cijothomas) [inline src/OpenTelemetry.Exporter.Zipkin/README.md:30]:
```suggestion
  [Batch or Simple exporting processor](https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/trace/sdk.md#built-in-span-processors)
```

Comment (cijothomas) [reply src/OpenTelemetry.Exporter.Zipkin/README.md:30]: same for other readme as well.

**Review by cijothomas — APPROVED** [review]

**Pull request merged**